Duplicate identifier `TriggerOptions`

`index.d.ts` has two declarations of the type `TriggerOptions`:

```ts
node_modules/node-pg-migrate/index.d.ts(204,13): error TS2300: Duplicate identifier 'TriggerOptions'.

204 export type TriggerOptions = TriggerOptionsEn & FunctionOptions
                ~~~~~~~~~~~~~~


node_modules/node-pg-migrate/index.d.ts(280,13): error TS2300: Duplicate identifier 'TriggerOptions'.

280 export type TriggerOptions = TriggerOptionsEn & FunctionOptions
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -277,8 +277,6 @@ export interface OperatorListDefinition {
     params?: FunctionParam[]
 }
 
-export type TriggerOptions = TriggerOptionsEn & FunctionOptions
-
 export interface MigrationBuilder {
     // Tables
     createTable(tableName: Name, columns: ColumnDefinitions, options?: TableOptions): void

